service/matching/task_queue_partition_manager.go

```diff
@@ -73,8 +74,10 @@ type (
 		HasPollerAfter(buildId string, accessTime time.Time) bool
 		// HasAnyPollerAfter checks pollers on all versioned and unversioned queues
 		HasAnyPollerAfter(accessTime time.Time) bool
-		// DescribeTaskQueue returns information about the target task queue
-		DescribeTaskQueue(includeTaskQueueStatus bool) *matchingservice.DescribeTaskQueueResponse
+		// LegacyDescribeTaskQueue returns information about the target task queue's pollers and the status of its default queue
+		LegacyDescribeTaskQueue(includeTaskQueueStatus bool) *matchingservice.DescribeTaskQueueResponse
+		// DescribeTaskQueuePartition returns information about the target task queue partition
+		DescribeTaskQueuePartition(request *matchingservice.DescribeTaskQueuePartitionRequest) (*matchingservice.DescribeTaskQueuePartitionResponse, error)
```

Comment:
should we call this only `Describe` too? the comment doesn't seem to say anything that is not obvious from the interface itself. so we can delete it.

Reply:
yeah I think that would be good